src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -127,11 +124,13 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
             return sb.ToString();
         }
 
-        Task StartEndpoints(IEnumerable<ComponentRunner> endpoints, CancellationTokenSource cts)
+        async Task StartEndpoints(IEnumerable<ComponentRunner> endpoints)
         {
-            var startTimeout = TimeSpan.FromMinutes(2);
-            return endpoints.Select(endpoint => StartEndpoint(endpoint, cts))
-                .Timebox(startTimeout, $"Starting endpoints took longer than {startTimeout.TotalMinutes} minutes.");
+            using var startTimeoutCts = CreateTimeoutCancellatioTaskCompletionSource(TimeSpan.FromMinutes(2));
+
+            await Task.WhenAll(endpoints.Select(endpoint => StartEndpoint(endpoint, startTimeoutCts)))
```

Comment:
the cts could be disposed after a timeout and then a starting endpoint might still try to access it but it looks all potential calls to the token are safe to call even on a disposed cts (only cancel in `StartEndpoint` would throw but that wouldn't do much as that exception will just be lost in the background)